backend/telemetry_core/src/main.rs

```diff
@@ -537,6 +537,10 @@ async fn return_prometheus_metrics(aggregator: AggregatorSet) -> Response<hyper:
             "telemetry_total_messages_to_feeds{{aggregator=\"{}\"}} {} {}\n",
             idx, m.total_messages_to_feeds, m.timestamp_unix_ms
         ));
+        s.push_str(&format!(
+            "telemetry_current_messages_to_aggregator{{aggregator=\"{}\"}} {} {}\n\n",
```

Comment:
Do you think it's worth ghetto-namespacing the label with "core" somewhere, so it's crystal clear what this refers to? Maybe `telemetry_core_current_messages_to_aggregator` or `telemetry_current_messages_to_core_aggregator`?

Reply:
That's a good idea; I guess I haven't done that yet because all of the telemetry is currently from/about the core